src/diffs/package-json.js

```diff
@@ -0,0 +1,37 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('lodash')
```

Comment:
For browser code, sure size argument makes sense. For backend, not so much imo. It's going to be pulled down regardless because it's used by so many dependencies (we already are pulling it - this code worked without me installed it explicitly). _.get/_.has makes code a lot more readable.  I agree that there are some more sophisticated function which make things less readable, but I don't think I used any of those here.